From 738f6f045e740c3fc21579297990d60b7c2e83ed Mon Sep 17 00:00:00 2001
From: Petr Gotthard <petr.gotthard@centrum.cz>
Date: Sat, 17 Jul 2021 20:23:32 +0200
Subject: Test: Remove duplicate openssl req -new
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The command is called twice, which is superfluous. Both the key
and the certificate are created already by the first command.

Signed-off-by: Petr Gotthard <petr.gotthard@centrum.cz>
---
 script/ekca/create_ca.sh | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/script/ekca/create_ca.sh b/script/ekca/create_ca.sh
index 90a5c791..61d976a3 100755
--- a/script/ekca/create_ca.sh
+++ b/script/ekca/create_ca.sh
@@ -94,11 +94,6 @@ ${SED_CMD} "s|ROOTCRT|$ROOT_URL|g" $OPENSSL_CONF
 
 openssl req -new -out intermed-ca.req.pem -passout file:pass.txt
 
-openssl req -new \
-    -key private/intermed-ca.key.pem \
-    -out intermed-ca.req.pem \
-    -passin file:pass.txt
-
 openssl rsa -inform PEM -in private/intermed-ca.key.pem \
         -outform DER -out private/intermed-ca.key.der -passin file:pass.txt
 
-- 
2.26.3

